docs/docs/basics/installation.mdx

```diff
@@ -62,11 +62,4 @@ cd ios && pod install
 
 ## Expo
 
-You can now use React Native Track Player with Expo.
-
-Start by creating a [custom development client](https://docs.expo.dev/clients/getting-started/) for your Expo app and then install React Native Track Player.
-
-Here is the configuration required for audio playback in background:
-
-- [iOS: Enable audio playback in background via your app.json](https://docs.expo.dev/versions/latest/sdk/audio/#playing-or-recording-audio-in-background-ios)
-- [Android: Stop playback when the app is closed](./background-mode.md/#android)
+Unfortunately, we do not support Expo at this time.
```

Comment:
@jcksncllwy this isn't entirely true, although neither is the earlier message. I know that there are a lot of users who are using Expo successfully, so I don't want to remove this documentation. However, it's probably good to add a note/warning that while it does work you can't expect much support from the maintainers (as they currently exist).

Reply:
Gotcha. I've updated the wording to better reflect that. Let me know what you think